reverse_engineering/connection_settings_modal/connectionSettingsModalConfig.json

```diff
@@ -112,5 +112,19 @@
 				}
 			}
 		]
+	},
+	{
+		"tab": "Client",
+		"structure": [
+			{
+				"inputLabel": "Client Path",
```

Comment:
must be optional, first look in the env variables, standard paths depending on OS, then require to set client path. This logic should be included in test connection

Reply:
$ORACLE_HOME ?